RocketChat/Rocket.Chat.ReactNative
# [FIX] 'Send to channel' when replying as a quote

## Proposed changes

`Send to channel` should show up only when replying as a thread.

## Issue(s)	


## How to test or reproduce

- Start a quote reply
- Send to channel option should be hidden

## Screenshots

## Types of changes




## Checklist



## Further comments
